msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -1557,6 +1568,14 @@ private static boolean isUuid(@NonNull final String tenantIdNameOrAlias) {
     public IAuthenticationResult acquireTokenSilent(
             @NonNull final AcquireTokenSilentParameters acquireTokenSilentParameters)
             throws InterruptedException, MsalException {
+        return acquireTokenSilentInternal(acquireTokenSilentParameters, PublicApiId.PCA_ACQUIRE_TOKEN_SILENT_WITH_PARAMETERS);
```

Comment:
We might want to know if this is invoked by MultipleAccountPCA or SingleAccountPCA.

i.e. It might be hard to tell which acquireToken/acquireTokenSilent overload in MutlipleAccountPCA is 'popular', since we can't tell how many % of the number we get from this PCA_ACQUIRE_TOKEN_SILENT_WITH_PARAMETERS is from MultipleAccountPCA.

Reply:
This means we would need to override this in the subclasses, and mark this as 'not implemented'.
(Might require some change in robolectric, though.)

Reply:
@rpdome Based on what we discussed, there are about four different methods for which we will have to do this. So that would result in creating 8 new methods (4 in each Single Account PCA & Multiple Account PCA) that all do the same thing. I prefer not to take this change at the moment. We can do this later once we find a better way to achieve this.